non-oss/addons/kontena-backup/addon.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+Pharos.addon 'kontena-backup' do
+  using Pharos::CoreExt::DeepTransformKeys
+
+  version '0.1.0'
```

Comment:
Just thinking should we follow pharos version here? Anyway `0.1.0` sounds a bit too unstable 😄

Reply:
Auto-bump:  `version: Pharos::VERSION` ?

Reply:
that then creates a problem on docs side, as we have the version in addon docs

Reply:
As discussed in Slack: `0.9.6+kontena.1`